app.rb

```diff
@@ -15,10 +15,15 @@ class App
   class << self
     def client
       DataSource::Client::Github.new(
-        host: ENV['GITHUB_HOST'] || 'api.github.com',
-        access_token: ENV['GITHUB_ACCESS_TOKEN'],
-        cache_dir: ENV['alfred_workflow_cache']
-      )
+          host: ENV['GITHUB_API_HOST'] || 'api.github.com',
+          access_token: ENV['GITHUB_ACCESS_TOKEN'],
+          me_account: ENV['GITHUB_ME_ACCOUNT'] || '@me',
+          pr_all_involve_me: bool_env('PR_ALL_INVOLVE_ME'),
+          cache_dir: ENV['alfred_workflow_cache'],
+          cache_ttl_sec_repo: (ENV['CACHE_TTL_SEC_REPO'] || (24 * 60 * 60)).to_i,
+          cache_ttl_sec_org: (ENV['CACHE_TTL_SEC_ORG'] || (24 * 60 * 60)).to_i,
+          cache_ttl_sec_pr: (ENV['CACHE_TTL_SEC_PR'] || (5 * 60)).to_i,
+          )
```

Comment:
See me other comment related to the configuration hash below and maybe we can move the defaults there as well.